### ISSUE TITLE
[BUG] Send commands as text messages doesn't work

### ISSUE THREAD

**Issue by PaSSw0rds** [blueset/efb-gpg-middleware#1]
**Describe the bug**  
Send commands as text messages in any channel or send texts to tg_bot ，still let me 「请回复一条过去的消息。」
**To Reproduce**  
1. enabled plugin
2. send text to bot

**Expected behavior**  

**Screenshots**  
no Screenshots

**Version information (please complete the following information):**  
- OS: Linux-5.4.17-2102.201.3.el7uek.aarch64-aarch64-with-oracle-7.9
- Detailed Python, EFB and module versions: (you can obtain this by running `ehforwarderbot --version`)

```
EH Forwarder Bot
Version: 2.1.1
Python version: 3.6.8

master_channel: blueset.telegram
slave_channels:
- blueset.wechat
middlewares:
- patch.PatchMiddleware
- blueset.gpg
```

**Additional context**  

发送消息后指令无法回应，无法测试是否以及启用
gpg的指纹和密码已经加入到config中
```
# Signature of your own key pair (required)
key: D97B9F32CC0ED067BFC8C41EC5A76F79C47*****

# Passphrase to the key pair (optional)
password: ******
```

**Comment by PaSSw0rds** [comment]
```
微信传输助手, [01.07.21 11:15]
您正在使用 EFB Telegram 主端 2.2.4，运行于默认配置档案与实例、EFB 版本 2.1.1。
已激活 1 个从端：
- 💬 WeChat Slave (blueset.wechat, 2.0.5)

已启用 2 个中间件：
- Patch Middleware (patch.PatchMiddleware, 2.0.17)
- GnuPG Middleware (blueset.gpg, 1.1.0)

myusername, [01.07.21 11:17]
gpg`info

微信传输助手, [01.07.21 11:17]
[回复 myusername]
错误：未指定消息接收者。
请回复一条过去的消息。
```

**Comment by blueset** [comment]
To use the <code>gpg`info</code> command, you need to send it to a specific remote chat (in your case, a WeChat user or group), so that GPG middleware can catch it. If this still doesn’t help, please provide your verbose log during reproduction of the issue.

使用 <code>gpg`info</code> 指令需要将其发送给远端会话（比如说某个微信用户或群组），以便 GPG 中间件可以截获指令。 如果问题仍然存在，希望能提供问题发生时的 verbose 日志。

**Comment by PaSSw0rds** [comment]
> To use the `` gpg`info `` command, you need to send it to a specific remote chat (in your case, a WeChat user or group), so that GPG middleware can catch it. If this still doesn’t help, please provide your verbose log during reproduction of the issue.
> 
> 使用 `` gpg`info `` 指令需要将其发送给远端会话（比如说某个微信用户或群组），以便 GPG 中间件可以截获指令。 如果问题仍然存在，希望能提供问题发生时的 verbose 日志。

```
private message (verbose log) has been removed by author
```

**Comment by blueset** [comment]
Thanks for the report. 
There was a discrepancy for the info command in README and the actual code. This is now fixed in version 1.1.1 to include both `info` and `show` commands as aliases to each other.

You may consider update, or use the command <code>gpg`show</code> on the current version.

**Comment by PaSSw0rds** [comment]
> Thanks for the report.
> There was a discrepancy for the info command in README and the actual code. This is now fixed in version 1.1.1 to include both `info` and `show` commands as aliases to each other.
> 
> You may consider update, or use the command `` gpg`show `` on the current version.

thx for your reply, i will try that

---
now works to my machine。test on 2021/07/04 12:23 GMT+0800

```
您正在使用 EFB Telegram 主端 2.2.4，运行于默认配置档案与实例、EFB 版本 2.1.1。
已激活 1 个从端：
- 💬 WeChat Slave (blueset.wechat, 2.0.5)

已启用 2 个中间件：
- Patch Middleware (patch.PatchMiddleware, 2.0.17)
- GnuPG Middleware (blueset.gpg, 1.1.1)
```
